cli/internal/packager/create.go

```diff
@@ -119,10 +119,16 @@ func addLocalAssets(tempPath componentPaths, assets config.ZarfComponent) {
 		// Load all specified git repos
 		for _, url := range assets.Repos {
 			matches := strings.Split(url, "@")
-			if len(matches) < 2 {
-				logrus.WithField("remote", url).Fatal("Unable to parse git url. Ensure you use the format url.git@tag")
+			// Pull all of the references if there is no `@` in the string
+			pullAll := len(matches) == 1
+			cloneDir := git.Pull(url, tempPath.repos, pullAll)
+			if !pullAll {
```

Comment:
I'm with with it either way but I think this logic would make sense to put at the end of the `git.Pull( ... )` function.

Reply:
Actually, the first implementation I had for this did look like that haha. I ended up switching to this because I wasn't sure of the scope of git pull usage for Zarf. I figured that it would be easier to implement it with no limitations (such as deleting the refs) so that if a pull were ever needed for a different reason, that didn't fetch all tags, a refactor wasn't needed. Hopefully that at least clears up my thought process for not including it directly in the pull. If `git.Pull` will only ever be needed for charts pulls and repo mirrors, I see no reason not include the tag checkout/reference deletion logic at the end, and if you'd like, I can make that happen while handling the README update.